parser/encoding/rpc.go

```diff
@@ -15,13 +15,22 @@ import (
 	schema "encr.dev/proto/encore/parser/schema/v1"
 )
 
+type Lang int
+
+const (
+	_ Lang = iota
+	GO
+	TypeScript
+)
+
```

Comment:
Are you meaning to add the languages back in here?

Reply:
Nope. Thanks for noticing. 😮‍💨